devicetypes/smartthings/zwave-panic-button.src/zwave-panic-button.groovy

```diff
@@ -0,0 +1,149 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+metadata {
+	definition(name: "Z-Wave Panic Button", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.remotecontroller") {
+		capability "Sensor"
+		capability "Health Check"
+		capability "Battery"
+		capability "Panic Alarm"
+
+		fingerprint mfr: "0258", prod: "0003", model: "108A", deviceJoinName: "NEO SOS"
+		fingerprint mfr: "0258", prod: "0003", model: "108F", deviceJoinName: "NEO SOS"
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name: "panic", type: "generic", width: 6, height: 4) {
+			tileAttribute("device.panicAlarm", key: "PRIMARY_CONTROL") {
+				attributeState "panic", label: "Emergency!", icon:"st.alarm.beep.beep", backgroundColor:"#bc2f2f"
+				attributeState "clear", label: "", icon:"st.Weather.weather13", backgroundColor:"#a4a9aa"
+			}
+		}
+
+		childDeviceTiles("buttons")
+
+		valueTile("battery", "device.battery", decoration: "flat", inactiveLabel: false, width: 2, height: 2) {
+			state "battery", label: '${currentValue}% battery', unit: ""
+		}
+
+		main(["panic"])
+
+		details(["panic", "buttons", "battery"])
+	}
+}
+
+def installed() {
+	log.debug "Installed $device.displayName"
+	sendEvent(name: "DeviceWatch-Enroll", value: [protocol: "zwave", scheme:"untracked"].encodeAsJson(), displayed: false)
+	if(hasMultipleButtons()) {
+		addChildButtons()
+	}
+	setPanicAlarm("clear")
+	def cmds = [
+			zwave.batteryV1.batteryGet().format(),
+			zwave.notificationV3.notificationGet().format(),
+			zwave.wakeUpV1.wakeUpNoMoreInformation().format()
```

Comment:
You should put a delay in before the `noMoreInformation` or the device might go to sleep before it reports the batteries and notification.